### ISSUE TITLE
Filename containing : creating issue in Windows

### ISSUE THREAD

**Issue by ArunTS96** [BennyThadikaran/eod2#205]
C:\Users\Admin\Documents\Stocks\Code\PythonRS\eod2\src\eod2_data>git pull  https://github.com/BennyThadikaran/eod2_data.git main
From https://github.com/BennyThadikaran/eod2_data
 * branch                  main       -> FETCH_HEAD
error: invalid path 'daily/nifty500 multicap 50:25:25.csv'
error: invalid path 'daily/nifty500 multicap india manufacturing 50:30:20.csv'
error: invalid path 'daily/nifty500 multicap infrastructure 50:30:20.csv'
Updating 7f92a1bca2..a39e058343

Windows cannot have `:` in file name

**Comment by BennyThadikaran** [comment]
Hi, thanks for bring this issue up.

In `src/defs/defs.py` there is a `updateIndice` function which check for `/` and replaces it with `-`.  Need to check for `:` as well.

I will update the code and files in eod2_data. Will post the update tomorrow morning.
Regards,
Benny Thadikaran

**Comment by BennyThadikaran** [comment]
This is fixed. I have tested by using setup_data.py in Window 10. The setup was successful though I havent run a full data sync.

I did notice some issues with Python 3.8 compatibility. For example Pandas 2.0.3 is the last supported version for Python 3.8.

I will fix the Python 3.8 compatibility in a separate follow up update. By today evening, I can complete the rest of the test on Windows and Linux.
